pkg/services/rendering/rendering.go

```diff
@@ -293,6 +315,28 @@ func (rs *RenderingService) RenderCSV(ctx context.Context, opts CSVOpts, session
 	return result, err
 }
 
+func (rs *RenderingService) SanitizeSVG(ctx context.Context, req *SanitizeSVGRequest) (*SanitizeSVGResponse, error) {
+	capability, err := rs.HasCapability(SvgSanitization)
+	if err != nil {
+		return nil, err
+	}
+
+	if !capability.IsSupported {
+		return nil, fmt.Errorf("svg sanitization unsupported, requires image renderer version: %s", capability.SemverConstraint)
```

Comment:
This ultimately returns an "upload internal error" to the user:
![image](https://user-images.githubusercontent.com/35176601/177340761-04ee6462-19b0-4db4-b31d-34fc0a133f63.png)

I think it would be better UX if we could return this error instead.

Reply:
we are completely replacing the upload popover in https://github.com/grafana/grafana/pull/50638 so I think we can leave it for now:) 

I want to have something similar to dashboard previews with the alert textbox - the user should not be able to press the upload button when the sanitization is required, and the service is not available.